win32ss/user/ntuser/timer.c

```diff
@@ -17,7 +17,9 @@ static LIST_ENTRY TimersListHead;
 static LONG TimeLast = 0;
 
 /* Windows 2000 has room for 32768 window-less timers */
-#define NUM_WINDOW_LESS_TIMERS   32768
+/* These values give timer IDs [256,32767], same as on Windows */
+#define NUM_WINDOW_MAX_TIMERS   (32768 - 1)
```

Comment:
That name doesn't really make sense. Use something like `MAX_WINDOW_LESS_TIMER_ID`